kuma/users/tests/test_views.py

```diff
@@ -1121,6 +1122,20 @@ def test_login_500_on_getting_email_addresses(self):
         doc = pq(resp.content)
         assert 'Account Sign In Failure' in doc.find('h1').text()
 
+    def test_login_SSLError_on_getting_profile(self):
```

Comment:
I'm not sure what these two new tests are testing, since they don't really test whether `KumaGitHubOAuth2Adapter.complete_login` uses `requests_retry_session()` or not (both complete successfully whether I use `requests_retry_session()` or not). Maybe you added them simple to more fully flesh-out these tests?

Reply:
I believe so. It's been a while but I think the intention was to just add some more coverage that didn't already exist.